test/libsolidity/syntaxTests/inheritance/override/public_vars_multiple.sol

```diff
@@ -0,0 +1,11 @@
+contract A {
+	function foo() external virtual pure returns(uint) { return 5; }
+}
+contract B {
+	function foo() external virtual pure returns(uint) { return 5; }
+}
+contract X is A, B {
+	uint public override foo;
```

Comment:
Does it work if you change it to `override(A, B)` or do we only allow a single base for such cases? (I'm fine with either)

Reply:
changing it still will fail

Reply:
Please add a test.